docs/nydus-fscache.md

```diff
@@ -58,6 +58,26 @@ $ sudo yum update kernel --enablerepo Plus
 $ sudo reboot
 ```
 
+## Enable fscache
+
+1.  ``[ -c /dev/cachefiles ] && echo ok`` to test fscache is enable or not. If your result shows `ok`, then fscache has been already enabled; otherwise, please follow the following steps to enable fscache.
+
+2.  Download cachefilesd package. For centos users, the command is:
+```
+sudo yum install cachefilesd
```

Comment:
Why is `cachefilesd` needed? Nydusd should act as `cachefilesd`in fscache mode.

Reply:
Oh, it's to install fscache kernel drivers. No need to start the `cachefilesd` daemon:)

Reply:
Starting `cachefilesd` daemon here is to generate the device file `/dev/cachefiles` which nydus needs. 
https://github.com/dragonflyoss/nydus/blob/ac55d7f932c62c662760b9bc191eccc26b2ee40c/service/src/singleton.rs#L140-L146